utils/check_modular_conversion.py

```diff
@@ -135,14 +135,32 @@ def guaranteed_no_diff(modular_file_path, dependencies, models_in_diff):
     skipped_models = set()
     non_matching_files = 0
     ordered_files, dependencies = find_priority_list(args.files)
-    for modular_file_path in ordered_files:
-        is_guaranteed_no_diff = guaranteed_no_diff(modular_file_path, dependencies, models_in_diff)
-        if is_guaranteed_no_diff:
-            model_name = modular_file_path.rsplit("modular_", 1)[1].replace(".py", "")
-            skipped_models.add(model_name)
-            continue
-        non_matching_files += compare_files(modular_file_path, args.fix_and_overwrite)
-        models_in_diff = get_models_in_diff()  # When overwriting, the diff changes
+    if args.fix_and_overwrite:
+        for modular_file_path in ordered_files:
+            is_guaranteed_no_diff = guaranteed_no_diff(modular_file_path, dependencies, models_in_diff)
+            if is_guaranteed_no_diff:
+                model_name = modular_file_path.rsplit("modular_", 1)[1].replace(".py", "")
+                skipped_models.add(model_name)
+                continue
+            non_matching_files += compare_files(modular_file_path, args.fix_and_overwrite)
+            models_in_diff = get_models_in_diff()  # When overwriting, the diff changes
+    else:
+        new_ordered_files = []
+        for modular_file_path in ordered_files:
+            is_guaranteed_no_diff = guaranteed_no_diff(modular_file_path, dependencies, models_in_diff)
+            if is_guaranteed_no_diff:
+                model_name = modular_file_path.rsplit("modular_", 1)[1].replace(".py", "")
+                skipped_models.add(model_name)
+            else:
+                new_ordered_files.append(modular_file_path)
+
+        new_ordered_files = ordered_files
+        import multiprocessing
+
+        with multiprocessing.Pool(4) as p:
```

Comment:
instead of `4`, should we set it to the number of available cores (or a multiple of it)?

Reply:
yes, I am too lazy as developer 😢